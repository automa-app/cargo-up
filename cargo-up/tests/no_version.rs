use insta::assert_snapshot;
mod utils;

#[test]
fn test_no_version() {
    let (out, _) = utils::run(
        "../fixtures/rename_member",
        &[
            "up",
            "dep",
            "upgradee",
            "--lib-path",
            "../../cargo-up",
            "--path",
            "../upgrader",
            "--name",
            "upgrader",
            "--to-version",
            "0.2.1",
        ],
    );
    assert_snapshot!(out);
}
