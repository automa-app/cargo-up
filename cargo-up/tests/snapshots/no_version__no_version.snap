---
source: tests/no_version.rs
expression: out
---
Upgrader for crate upgradee has not described any changes for 0.2.1 version
